modular_skyrat/master_files/code/modules/clothing/under/akula_jobs.dm

```diff
@@ -0,0 +1,302 @@
+/// How many wetstacks does the clothing's status effect apply to its wearer
+#define STATUS_EFFECT_STACKS 15
+
+/// A proc to be used in `equipped()` for all akula clothing which has the 'special tech' to keep their wearers slippery
+/obj/item/clothing/proc/apply_wetsuit_status_effect(mob/living/carbon/human/user)
+	if(!HAS_TRAIT(user, TRAIT_SLICK_SKIN))
+		return FALSE
+	user.apply_status_effect(/datum/status_effect/wetsuit)
+
+/// A proc to remove the wetsuit status effect, used with the `dropped()` proc
+/obj/item/clothing/proc/remove_wetsuit_status_effect(mob/living/carbon/human/user)
+	if(!HAS_TRAIT(user, TRAIT_SLICK_SKIN))
```

Comment:
Also, it should probably remove the status effect even if they don't have the trait slick skin, just for consistency, and for future-proofing. I don't think it'll runtime if it tries to remove a status effect that the user doesn't have (needs testing though).